### ISSUE TITLE
Keyboard navigation doesn’t work as expected when using RTL

### ISSUE THREAD

**Issue by jouni** [vaadin/vaadin-radio-button#70]
The left and right arrow keys should move the focus in a different direction when using RTL.

**Comment by web-padawan** [comment]
Fixed by #128. Will be released as a minor because of related `vaadin-element-mixin` bump.
